src/main/java/edu/ucsb/cs156/organic/services/SystemInfoServiceImpl.java

```diff
@@ -23,13 +23,26 @@ public class SystemInfoServiceImpl extends SystemInfoService {
   private boolean showSwaggerUILink;
 
   @Value("${app.sourceRepo}")
-  private String sourceRepo = "https://github.com/ucsb-cs156/proj-organic";
+  private String sourceRepo = "https://github.com/ucsb-cs156-s24/proj-organic-s24-5pm-4";
+
+  @Value("${git.commit.message.short:unknown}")
+  private String commitMessage;
+
+  @Value("${git.commit.id.abbrev:unknown}")
+  private String commitId;
+
+  public static String githubUrl(String repo, String commit) {
+    return commit != null && repo != null ? repo + "/commit/" + commit : null;
+  }
 
   public SystemInfo getSystemInfo() {
     SystemInfo si = SystemInfo.builder()
     .springH2ConsoleEnabled(this.springH2ConsoleEnabled)
     .showSwaggerUILink(this.showSwaggerUILink)
-    .sourceRepo(this.sourceRepo)
+    .sourceRepo("https://github.com/ucsb-cs156-s24/proj-organic-s24-5pm-4")
+    .commitMessage(this.commitMessage)
+    .commitId(this.commitId)
+    .githubUrl(githubUrl("https://github.com/ucsb-cs156-s24/proj-organic-s24-5pm-4", this.commitId))
```

Comment:
```suggestion
    .githubUrl(githubUrl( this.sourceRepo, this.commitId))
```
Again, no hard coding.